build/BuildEnv.shade

```diff
@@ -33,4 +33,12 @@ functions
             return Environment.GetEnvironmentVariable("TEAMCITY_PROJECT_NAME") != null;
         }
     }
-  }
+    
+    bool IsBuildV2
+    {
+        get 
+        { 
+            return Environment.GetEnvironmentVariable("KOREBUILD_BUILD_V2") == "1"; 
```

Comment:
That's a temporary variable that allows me to enable the new build for only a few repos. I plan to remove it in a week or so once the build is stable. But I don't want to affect the internal build while making these changes